apps/wallet-dashboard/components/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import { PropsWithChildren, useState } from 'react';
+import {
+    AccordionHeaderProps,
+    Accordion,
+    AccordionHeader,
+    AccordionContent,
+    TitleSize,
+    Title,
+} from '@iota/apps-ui-kit';
+
+type CollapsibleProps = {
+    title: string;
+    hideBorder?: boolean;
+    defaultOpen?: boolean;
```

Comment:
You might as well call it `defaultExpanded` instead of `open` so it follows the design of the other accordion components